src/snowflake/snowpark/udf.py

```diff
@@ -85,6 +86,10 @@ def __call__(
                     f"The input of UDF {self.name} must be Column, column name, or a list of them"
                 )
 
+        session = snowflake.snowpark.context.get_active_session()
```

Comment:
@sfc-gh-aalam This line may raise an exception when there are multiple active sessions.